backend/src/main/java/com/google/minicrm/servlets/FormsServlet.java

```diff
@@ -113,6 +110,19 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response)
     response.setStatus(204); //Success - 204 No Content
   }
 
+  /**
+   * Converts an array of Strings to their Long representation
+   * @param numbers array of Strings representing numbers
+   * @return        array of Longs
+   */
+  private long[] convertLongArrToStringArr(String[] numbers) {
+    Long[] result = new Long[numbers.length];
```

Comment:
I think this can just be primitive long, as the value of Long.parseLong should still be unboxed to primitive long.

Reply:
fixed